src/frontend/src/lib/components/carousel/Carousel.svelte

```diff
@@ -114,6 +115,9 @@
 	 * Start autoplay timer
 	 */
 	const initialiseAutoplayTimer = () => {
+		
+		if (JSON.parse(import.meta.env.VITE_DISABLE_CAROUSEL_AUTO_PLAY ?? false) === true) return
```

Comment:
Can you move the parsing to another file? just for mantainance

Reply:
done